packages/manager/apps/telecom/src/app/telecom/pack/move/offers/move-offers.controller.js

```diff
@@ -210,10 +225,54 @@ export default class PackMoveOffersCtrl {
     set(offer, 'displayedPrice', displayedPrice);
   }
 
+  static updateSelectedGtrOption(value, offer, optionName) {
+    angular.forEach(offer.options, (option) => {
+      if (
+        option.name.match(/^gtr_/) &&
+        option.selected !== null &&
+        optionName.match(/^gtr_/)
+      ) {
+        if (option.name !== optionName) {
+          option.selected = false; // eslint-disable-line no-param-reassign
+        }
+      }
+    });
+  }
+
   static isChosen(option) {
     return option.choosedValue > 0;
   }
 
+  static isGtrOption(optionName) {
+    return optionName.match(/^gtr_/) ? 1 : 0;
+  }
+
+  static isGtrOptionExist(offer) {
+    let gtr = false;
+    angular.forEach(offer.options, (option) => {
```

Comment:
**question:** Couldn't `Array.prototype.some` be used?

Reply:
done thx